docs/README.md

```diff
@@ -6,86 +6,140 @@
 
-## Functionalities
+When navigating a B2B store as a customer, it is common for a main user from an organization to have other people under their structure, each with their own information and access privileges.
 
-- Centralize App Permissions
-- Manage Roles Permissions
-- Impersonation
+Within an organization, each user can have different roles, such as a professional buyer that places orders with budget limits from a predefined cost center or  a manager in charge of reviewing and approving orders. These roles can be associated with multiple permissions, depending on the actions this user needs to perform.
 
-## Configuration
+The **Storefront Permissions** app stores a predefined set of roles and app permissions related to what B2B users can access and do in your storefront, making this information available for other integrated apps to check. This is useful for stores who want to set specific app permissions for users with different roles in an organization.
 
-1. [Install](https://vtex.io/docs/recipes/development/installing-an-app/) the storefront-permissions app by running `vtex install vtex.storefront-permissions` in your terminal.
-2. At the Admin, navigate to **ACCOUNT SETTINGS > Storefront Permissions**.
 
-3. Manage Roles
+## Available storefront roles
 
-## Advanced app integration
+In the following table, you can see the available storefront roles, their key used for identification in the app’s code, and their description.
 
-How to integrate your app with **Storefront Permissions** and make it available within the Role management
+| **Role** | **Key** | **Description** |
+|---|---|---|
+| Store Admin | `store-admin` | Store administrator, that is, a user who has access to the VTEX Admin. |
+| Sales Admin | `sales-admin` | Sales administrator user who can manage all sales users. |
+| Sales Manager | `sales-manager` | Sales manager user who can manage sales users in the same organization, as well as assist or impersonate buyers during navigation or purchase. |
+| Sales Representative | `sales-representative` | Sales representative user who can assist or impersonate buyers during navigation or purchase. |
+| Organization Admin | `customer-admin` | Main organization user who manages the organization information, as well as its members and cost centers. |
+| Organization Approver | `customer-approver` | Organization user who is in charge of reviewing orders placed by buyers, with the ability to approve them or deny them. |
+| Organization Buyer | `customer-buyer` | Organization user who has the ability to place orders in your store. |
```

Comment:
The information about `Approver` and `Buyer` is not accurate. Despite their name, by default the `Buyer` cannot place orders. They are able to add items to cart, and, if the B2B Quotes app is installed, they are able to save their cart as a saved cart or quote.

`Approvers` then are the ones who can take a saved cart (or quote) that was created by a `Buyer` and use it to place an order.

Reply:
I've changed it to the following:

| | | |
|---|---|---|
| Organization Approver | `customer-approver` | Organization user who can take a saved cart or quote that was created by an **Organization Buyer** and use it to place an order. |
| Organization Buyer | `customer-buyer` | Organization user who has the ability to to add items to cart. If the **B2B Quotes** app is installed, they are also able to save their cart for future use or create a quote. |